src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -219,5 +273,85 @@ public partial class VariableView : UserControl, IDisposable {
                     return KnownMonikers.BulletList;
             }
         }
+        #endregion
+
+        #region ICommandTarget
+        public CommandStatus Status(Guid group, int id) {
```

Comment:
Do you need a grid or a model from the grid?

---
In reply to: [67012100](https://github.com/Microsoft/RTVS/pull/1912#discussion_r67012100) [](ancestors = 67012100)